_pages/en_US/installing-boot9strap-(kartdlphax).txt

```diff
@@ -61,7 +64,8 @@ On the **target 3DS** (the 3DS that you are trying to modify):
 1. Launch Mario Kart 7
   - Ensure that wireless connectivity is enabled
   - kartdlphax should tell you that it is running
-1. Navigate to `Local Multiplayer` -> `Create Group`
+1. Select `Local Multiplayer`, then select your target console type and `3DS ROP xPloit Injector`, then `Use Settings`
```

Comment:
type -> model, maybe?

Reply:
it asks for the target console type where you have O3DS family and N3DS family, it doesnt ask for the model, if you mean that

Reply:
Might be good to separate this into multiple steps then? Would allow clarification on what "target console type" is meant to mean ("whether the console you are trying to install CFW on is an old 3ds or a new 3ds")

Reply:
i believe its pretty self-explanatory when you have 2 options to select, Old 3DS family and New 3DS family
![IMG_20220916_130549](https://user-images.githubusercontent.com/88926852/190625643-af1f1516-5856-4241-be34-f48d149c0782.jpg) I dont really understand your point here. Do you want to add something like "If you have a C-Stick above the ABYX buttons, you have a console from the New 3DS family, otherwise not?

Reply:
It's ultimately not super important (we can always change it later), but I personally want to emphasize the distinction between source console and target console. "Target console type" on its own might not be descriptive enough to fully convey "model of the 3DS that you are trying to mod". Maybe just being overly cautious - just leave it as-is for now